app/src/main/java/com/github/capntrips/kernelflasher/ui/screens/slot/SlotViewModel.kt

```diff
@@ -512,7 +520,7 @@ class SlotViewModel(
                     val updateBinary = File(akHome, "update-binary")
                     Shell.cmd("unzip -p \"$zip\" META-INF/com/google/android/update-binary > $akHome/update-binary").exec()
                     if (updateBinary.exists()) {
-                        val result = Shell.Builder.create().setFlags(Shell.FLAG_MOUNT_MASTER or Shell.FLAG_REDIRECT_STDERR).build().newJob().add("AKHOME=$akHome \$SHELL $akHome/update-binary 3 1 \"$zip\"").to(flashOutput).exec()
+                        val result = Shell.Builder.create().setFlags(Shell.FLAG_MOUNT_MASTER or Shell.FLAG_REDIRECT_STDERR).build().newJob().add("AKHOME=$akHome /system/bin/sh $akHome/update-binary 3 1 \"$zip\"").to(flashOutput).exec()
```

Comment:
Is there a need for this change?

Reply:
In some cases there may be no $SHELL environment variable

Reply:
A quick poke around the AOSP source and I see that path hardcoded, so I'm good with the change, if you don't mind breaking it off into its own commit.

Reply:
I will revert this and make another PR